browser/js/productDetail/productDetail.controller.js

```diff
@@ -0,0 +1,66 @@
+app.controller('ProductDetail', function($scope, ProductFactory, $stateParams, ReviewFactory, UserFactory, singleProduct, OrderFactory) {
+
+	$scope.product = singleProduct;
+
+	ReviewFactory.getProductReviews($stateParams.productId)
```

Comment:
This looks like some pretty heavy parsing, it should probably happen in the factory